operator/v1/types_network.go

```diff
@@ -100,12 +100,34 @@ type DefaultNetworkDefinition struct {
 	KuryrConfig *KuryrConfig `json:"kuryrConfig,omitempty"`
 }
 
+// SimpleMacvlanConfig contains configurations for macvlan interface.
+type SimpleMacvlanConfig struct {
+	// master is the host interface to create the macvlan interface from.
+	// If not specified, it will be default route interface
+	// +optional
+	Master string `json:"master,omitempty"`
+
+	// ipam indicates which IPAM module will be used for IP Address Management (IPAM)
+	// now supports 'dhcp'. If it's not specified does it default to dhcp.
+	// +optional
+	IPAM string `json:"ipam,omitempty"`
+
+	// mode is the macvlan mode: bridge, private, vepa, passthru. The default is bridge
+	// +optional
+	Mode MACVLANMode `json:"mode,omitempty"`
```

Comment:
Please be consistent with naming scheme, here you have `MACVLANMode` the struct is `SimpleMacvlanConfig` each time `Macvlan` has different case, please unify all of them.

Reply:
Fixed.